components/migrate/steps/SignLoader3.vue

```diff
@@ -1,5 +1,27 @@
 <template>
   <div class="mb-5">
+    <div class="flex items-center mb-4">
+      <div class="mr-5">
+        <NeoIcon
+          v-if="fromNfts?.length === nextNfts?.length"
+          v-bind="iconSuccess"
+          class="fa-2x" />
+        <NeoIcon v-else v-bind="iconIdle" class="fa-2x" />
```

Comment:
size prop?

Reply:
somehow "large" prop didn't work, and "medium" still `smaller` than `fa-2x`